### DISCUSSION TITLE
Support for multiple arbitrary box shadows

### DISCUSSION THREAD

**Discussion by jordanbaron** [tailwindlabs/tailwindcss#9290] [Ideas]
Right now we can create arbitrary box shadows with `shadow-[10px_10px]`. It would be useful to be able to create multiple shadows by doing `shadow-[10px_10px,-10px,-10px]`

**Comment by jordanbaron** [comment]
turns out this does work, not sure what i was doing wrong earlier. my apologies :/

**Reply by smithliam** [reply]
Yes, you can write any CSS property value, you just need to fill the spaces with an underscore `_`. No whitespace allowed!

```html
<div class="shadow-[10px 10px]"
```

This is wrong because `shadow-[10px` and 10px] are two different classes. I sometimes make the mistake and add a space...
